manifests/linux/redhat.pp

```diff
@@ -58,10 +58,17 @@
     require   => File['/etc/sysconfig/iptables'],
   }
 
+  # Redhat 7 selinux user context for /etc/sysconfig/iptables is set to unconfined_u
+  case $::selinux {
+    "${::lintminusminus}true",true: { $seluser = 'unconfined_u' }
```

Comment:
Could we use a [control comment](http://puppet-lint.com/controlcomments/) here instead?